charts/rancher-vsphere-csi/templates/node/windows-daemonset.yaml

```diff
@@ -5,6 +5,8 @@ apiVersion: apps/v1
 metadata:
   name: vsphere-csi-node-windows
   namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "labels" . | nindent 4 }}
```

Comment:
this comment is for this file, not these two lines:
 we need to add the `.Values.csiNode.podLabels` to the pod spec, right?

Reply:
removed include in windows as I don't have any windows machines to test on anyways.